lib/pub_sub.ex

```diff
@@ -63,6 +63,13 @@ defmodule PubSub do
   @doc """
   Returns a list of pids representing the processes that are currently
   subscribed to the given topic.
+
+  ## Example
+
+      iex> pid = self()
+      iex> PubSub.subscribe(pid, :my_topic)
+      iex> PubSub.subscribers(:my_topic) |> Enum.count
+      1
```

Comment:
this isn't very explicit. In reality `PubSub.subscribers` returns [#PID<123.1>] or something like that but I am unsure of how to match against the pid type in an array.

Reply:
`Enum.count` is good enough, but this would be better:

```elixir
iex> pid = self()
iex> PubSub.subscribe(pid, :my_topic)
iex> [subscriber] = PubSub.subscribers(:my_topic)
iex> subscriber == pid
true
```